### PR TITLE
issue-352 : Perform case insensitive match for IS_CONSUL_CLUSTER

### PR DESCRIPTION
**Pull request by bharath-b-hpe**
Consul entry point script used for starting consul service in container checks if IS_CONSUL_CLUSTER is set to enable consul cluster, which should be case insensitive match of the value.

### CODE REVIEW

**Review by jeevan-kamkar — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

File: install/Docker/dockerfiles/scripts/consul-entrypoint.sh [hunk 1]
```diff
@@ -13,7 +13,6 @@ set -e
 
 # If deployment type is HA mode, need to add the other consul instances
 # to form the quorum
-
```
Comment (Bharath-KKB) [inline install/Docker/dockerfiles/scripts/consul-entrypoint.sh:16]: whats the change ?

**Review by Bharath-KKB — COMMENTED** [review]
Whats the change?

File: install/Docker/dockerfiles/scripts/consul-entrypoint.sh (hunk 1 above)
Comment (bharath-b-hpe) [reply install/Docker/dockerfiles/scripts/consul-entrypoint.sh:16]: Expecting IS_CONSUL_CLUSTER to have true or false set, by including ",," value set is converted to lowercase and is used for making case insensitive match.

Comment (bharath-b-hpe) [reply install/Docker/dockerfiles/scripts/consul-entrypoint.sh:16]: And also deleted am empty line, for github to retrigger actions, as it was hung for long time and re-run wasn't available.

**Pull request merged**